### PR TITLE
Em/add new user fields

### PR DESCRIPTION
**Pull request by greatnessmensah**

### CODE REVIEW

File: api/api_models/user.py [hunk 1]
```diff
@@ -85,20 +104,25 @@ class ProfileUpdate(BaseModel):
     email: Optional[EmailStr]
     first_name: Optional[str]
     last_name: Optional[str]
+    yoe: Optional[int]
```
Comment (blitzblade) [inline api/api_models/user.py:107]: what's "yoe"

File: api/routes/tags.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    db_query =  db.query(User).filter(User.id == user.id).\
+        options(joinedload(User.tags)).first()   
+    return {"tags": db_query.tags}
+
+@tag_route.post('/tags', response_model=Tags, status_code=status.HTTP_201_CREATED)
+def create_tag(tag: TagCreate, current_user=Depends(get_current_user), db:Session=Depends(get_db)):
+    if not current_user:
+        raise HTTPException(status_code=401, detail='Unauthorized')
+
+    user_tag = db.query(Tag).filter(Tag.name == Tag.name).first()
```
Comment (blitzblade) [inline api/routes/tags.py:26]: `filter(Tag.name == Tag.name).first()`

What's being compared here? I don't get it. The field name is equal to what?

File: db/models/users.py [hunk 3]
```diff
@@ -15,6 +15,9 @@ class User(Base):
     last_name = Column(String, nullable=False)
     role_id = Column(Integer, ForeignKey("roles.id"), nullable=True)
     password = Column(String, nullable=False)
+    yoe = Column(Integer, nullable=False)
```
Comment (blitzblade) [inline db/models/users.py:18]: what's this? I don't understand and I'm not sure it's "self explanatory" enough 🤔

Comment (greatnessmensah) [reply db/models/users.py:18]: years of experience

File: api/routes/tags.py (hunk 2 above)
Comment (greatnessmensah) [reply api/routes/tags.py:26]: my bad, it was supposed to be filter(Tag.name == tag.name).first()

Comment (greatnessmensah) [reply api/routes/tags.py:26]: both works

Comment (blitzblade) [reply api/routes/tags.py:26]: If it works then cool

File: api/api_models/user.py (hunk 1 above)
Comment (blitzblade) [reply api/api_models/user.py:107]: Okay but make `years_of_experience`

Comment (greatnessmensah) [reply api/api_models/user.py]: done that

**Review by blitzblade — APPROVED** [review]

**Pull request merged**